src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp

```diff
@@ -244,6 +249,22 @@ const auto fusingScaleShift = fusingSpecificParams{ std::make_shared<postNodesMg
                 return std::make_shared<ngraph::opset1::Add>(cfg.input, constNode);
             }, "Add(PerChannel)"}}), {"Add"} };
 
+const auto fusingMultiEltwise = fusingSpecificParams{ std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
```

Comment:
I would say name "fuseClampRoundAddRelu" should be more convenient from developer standpoint